src/applications/vaos/tests/covid-19-vaccine/components/ReviewPage.unit.spec.jsx

```diff
@@ -182,7 +182,7 @@ describe('VAOS vaccine flow <ReviewPage>', () => {
     );
 
     // Not sure of a better way to search for test just within the alert
-    const alert = screen.baseElement.querySelector('.usa-alert');
```

Comment:
Do the usa-alert classes still exist on the page? We have at least one component (the review page) which searches for .usa-alert-error to find the alert to focus on.

Reply:
Only if the alert is background only, which in the case of the review page they are not. 

Ideally I would have updated the `scrollAndFocus` method to check and query the `shadowRoot` as well, but it looks like it's not able to query that node because it's not hydrated at the time of query. I added an `info-alert` class to focus on for those pages.